core/src/main/scala/spinal/core/Latch.scala

```diff
@@ -0,0 +1,23 @@
+package spinal.core
+
+object Latch {
+
+  def apply[T <: Data](dataType: HardType[T]): T = {
+    dataType.apply().addTag(noLatchCheck)
```

Comment:
```suggestion
    dataType().addTag(noLatchCheck)
```

You can just `apply` with `()`. This is done with `HardType` at other places in the code and I think it is better to remain consistent and use just `()`.

Reply:
I recall I had the `apply()` because IntelliJ underlined it for some reason. I'll drop it